pkgs/tools/security/tpm2-abrmd/default.nix

```diff
@@ -1,24 +1,36 @@
 { stdenv, fetchurl, lib
-, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
+, makeWrapper, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
 
 stdenv.mkDerivation rec {
   pname = "tpm2-abrmd";
-  version = "2.2.0";
+  version = "2.3.2";
 
   src = fetchurl {
```

Comment:
`fetchFromGitHub` would be cool, then we don't need to manually interpolate such a complicated URL.